templates/common/sdk_delete_post_request.go.tpl

```diff
@@ -0,0 +1,8 @@
+    if err == nil {
```

Comment:
ok with keeping in common for now but it has a dependency for requeueWaitWhileDeleting to be defined